web/src/components/Feedback.tsx

```diff
@@ -35,7 +35,12 @@ const TextInput = styled.input.attrs({
   resize: none;
 `
 
-export const Description = styled.div`
+export const Description = styled.label`
+  padding: 10px 0 5px;
+  font-weight: 700;
+`
+
+export const ErrorSendingStatus = styled.div`
```

Comment:
You could just use the `as` prop for this: https://styled-components.com/docs/basics#extending-styles

Reply:
@klinzo thx for the hint. does this make any difference in this case?